django_tenants/urlresolvers.py

```diff
@@ -22,16 +22,25 @@ def reverse(viewname, urlconf=None, args=None, kwargs=None, current_app=None):
 
 class TenantPrefixPattern:
     converters = {}
+    cached_domain = None
 
     @property
     def tenant_prefix(self):
         _DomainModel = get_tenant_domain_model()
         subfolder_prefix = get_subfolder_prefix()
         try:
-            domain = _DomainModel.objects.get(
-                tenant__schema_name=connection.schema_name,
-                domain=connection.tenant.domain_subfolder,
-            )
+            # Store the domain to avoid multiple DB hits for the same domain in the same request
+            if self.cached_domain is not None \
```

Comment:
PEP8 recommends avoiding backslash line continuation in Python because whitespace to the right of the backslash breaks the script on a change that is invisible to the reader.  Use parentheses instead.  https://peps.python.org/pep-0008/#maximum-line-length

Reply:
Thanks for the link! Been trysting the PyCharm auto-formatter a bit too much there.

I have refactored that bit to conform to pep8 standards now.